maven.jaxws/licenseinfo.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+ <!--
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+-->
+ <licenseinfo>
+    <fileset>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/error-badge.gif</file>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/waiting.png</file>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/webservice.png</file>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/webservicegroup.png</file>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/wsoperation.png</file>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/wsport.png</file>
+        <file>src/org/netbeans/modules/maven/jaxws/resources/XMLServiceDataIcon.png</file>
+        <license ref="Apache-2.0-ASF" />
+        <comment type="COMMENT_UNSUPPORTED" />
+    </fileset>
+</licenseinfo>
```

Comment:
Please add a new line, if possible.

Reply:
@junichi11 , Thanks for the review. I have added new lines in the latest commit.

Reply:
Thank you for fixing it :)